test/e2e/suites/import-gitops/import_gitops_test.go

```diff
@@ -121,34 +121,6 @@ var _ = Describe("[AWS] [EKS] Create and delete CAPI cluster functionality shoul
 	})
 })
 
-var _ = Describe("[Azure] [AKS] Create and delete CAPI cluster functionality should work with namespace auto-import", Label(e2e.DontRunLabel), func() {
```

Comment:
Can you elaborate more why is this redundant?

Reply:
Since we are using mgmtv3 cluster for CAPI cluster registration, we don’t need to test import scenarios for AKS using v2 cluster, as it has no added value anymore.